docs/website/docusaurus.config.js

```diff
@@ -93,7 +98,7 @@ module.exports = {
           ],
         },
       ],
-      // copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
+       copyright: `Copyright © ${new Date().getFullYear()} Odo Authors -- All Rights Reserved <br> Apache License 2.0 open source project`,
```

Comment:
I don't know if we should be using "odo" vs. "Odo" in the copyright. But I might be nitpicking because it's taken us some back and forth in the past to decide upon odo vs. Odo.
```suggestion
       copyright: `Copyright © ${new Date().getFullYear()} odo Authors -- All Rights Reserved <br> Apache License 2.0 open source project`,
```

Reply:
long time back we decided that it is "odo", we should use it everywhere